exercises/en/test_01_09a.py

```diff
@@ -8,6 +8,11 @@ def test():
     # Since we haven't started assigning charts to variable names yet,
     # this might be the better way to test for the first exercise.
     # Maybe even for later exercises.
+    assert 'from vega_datasets import data' in __solution__, "Did you import the correct library for data from vega_datasets?"
```

Comment:
Be careful here. Sometimes the students are not careful with white spaces.

Reply:
I would instead just check that "data" or "import data"  is in `__solution__`.  Agreed that white space is frustrating for students and checking less, is more robust sometimes.